src/OpenTelemetry/Metrics/HistogramConfiguration.cs

```diff
@@ -23,12 +23,13 @@ public class HistogramConfiguration : MetricStreamConfiguration
         private Aggregation aggregation = Aggregation.Histogram;
 
         /// <summary>
-        /// Gets or sets the custom histogram bounds.
+        /// Gets or sets the values representing explicit histogram bucket
+        /// boundary values.
         /// </summary>
         /// <remarks>
         /// The array must be in ascending order with distinct values.
         /// </remarks>
-        public double[] BucketBounds { get; set; }
+        public double[] Boundaries { get; set; }
```

Comment:
Consider changing this class name to `ExplicitBucketHistogramConfiguration` (so that later if we add exponential bucket histogram, we don't expose `Boundaries`).